src/core/PancakeInfinity.sol

```diff
@@ -80,14 +80,7 @@ interface IPancakeInfinityPoolManager {
     function poolIdToPoolKey(PoolId id)
         external
         view
-        returns (
-            IERC20 currency0,
-            IERC20 currency1,
-            IHooks hooks,
-            IPancakeInfinityPoolManager poolManager,
-            uint24 fee,
-            bytes32 parameters
-        );
+        returns (PoolKey memory);
```

Comment:
Are you sure about this?

Reply:
yep, we can do so for simplicity

Reply:
I agree that these are ABI-compatible, but what I'm saying is that it's probably preferable to do it the first way because it doesn't allocate. also it gives the compiler the opportunity to avoid loading values that we don't use

Reply:
Fair enough but, maybe we should just get rid of that interface. It is not used in the contract at the moment is just used in the tests that is why I decided to change it 😅. I can remove it there and move to the test. In the tests the above arguments are not that relevant